pypdf/_page_labels.py

```diff
@@ -0,0 +1,160 @@
+"""
+Page labels are shown by PDF viewers as "the page number".
+
+A page has a numeric index, starting with 0. Additionally to that, the page
+has a label. In the most simple case:
+    label = index + 1
+
+However, the title page and the table of contents might have roman numerals as
+page label. This makes things more complicated.
+
+Example 1
+---------
+
+>>> reader.trailer["/Root"]["/PageLabels"]["/Nums"]
+[0, IndirectObject(18, 0, 139929798197504),
+ 8, IndirectObject(19, 0, 139929798197504)]
+>>> reader.get_object(reader.trailer["/Root"]["/PageLabels"]["/Nums"][1])
+{'/S': '/r'}
+>>> reader.get_object(reader.trailer["/Root"]["/PageLabels"]["/Nums"][3])
+{'/S': '/D'}
+
+Example 2
+---------
+The following example shows a document with pages labeled
+i, ii, iii, iv, 1, 2, 3, A-8, A-9, ...
+
+1 0 obj
+    << /Type /Catalog
+    /PageLabels << /Nums [
+            0 << /S /r >>
+            4 << /S /D >>
+            7 << /S /D
+            /P ( A- )
+            /St 8
+            >>
+            % A number tree containing
+            % three page label dictionaries
+        ]
+        >>
+    ...
+    >>
+endobj
+
+
+PDF Specification 1.7
+=====================
+
+Table 159 – Entries in a page label dictionary
+----------------------------------------------
+The S-key:
+D       Decimal arabic numerals
+R       Uppercase roman numerals
+r       Lowercase roman numerals
+A       Uppercase letters (A to Z for the first 26 pages,
+                           AA to ZZ for the next 26, and so on)
+a       Lowercase letters (a to z for the first 26 pages,
+                           aa to zz for the next 26, and so on)
+"""
+
+from typing import Iterator
+
+from ._protocols import PdfReaderProtocol
+
+
+def number2decimal(number: int) -> str:
+    return str(number)
```

Comment:
I feel like this is an extraneous function to have, and could just do `str(...)` inline instead of having this function call, and that I wouldn't expect a function with the suffix `2decimal` to return a string to be honest.

Reply:
Good point - I'll remove it